iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -235,19 +237,14 @@ def createViewer(self, scriptUrl="js/dist/ixbrlviewer.js"):
 
             if f.context.isForeverPeriod:
                 aspects["p"] = "f"
-            elif f.context.isInstantPeriod and f.context.instantDatetime is not None:
+            elif f.context.isInstantPeriod:
```

Comment:
Why this change (and the `isStartEndPeriod` below)?  This appears to revert the fixes from #70.

Reply:
stale copy, put back in